andela/blade-runner-cfh
# #153741755 Users should receive a JWT on successful login

#### What does this PR do?
>- This PR ensures a user receives a JSON web token on successful sign-in
#### Description of Task to be completed?
>- Users once authenticated via the login api should be returned a JWT to be required on subsequent call to all other endpoints that require authentication.
#### How should this be manually tested?
>- Open postman, input the URL address which is "localhost:3000/api/users/signin"
>- input the following parameters:
    email: 'string'
    password: 'string'
>- send the request to the server
>- you should get a response containing the JSON web token

#### Any background context you want to provide?
>- N/A

#### What are the relevant pivotal tracker stories?
>- https://www.pivotaltracker.com/n/projects/2137297/stories/153741755

#### Screenshots (if appropriate):
<img width="1141" alt="screen shot 2018-01-14 at 1 15 37 am" src="https://user-images.githubusercontent.com/22421283/34911319-95dd13e2-f8c8-11e7-87d7-07ace7924f51.png">


#### Questions:
>- N/A
